### ISSUE TITLE
Compose button is not keyboard accessible

### ISSUE THREAD

**Issue by steverep** [mailvelope/mailvelope#602]
I'm trying to use the Firefox extension with Gmail, but the compose button is not keyboard accessible, i.e. it cannot be tabbed to and it does not seem to have a proper label.  This makes composing encrypted messages impossible for people unable to use a mouse or using an assistive technology like a screen reader.
